releasenotes/notes/28344.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: release-notes/v2
+kind: feature
+
+area: pilot
```

Comment:
Lgtm in general. One thing need to confirm with @brian-avery is that the `area: pilot`.

Hi @brian-avery , according to the release notes README: 
The listed `valid` areas do not include `pilot`, also, I checked the tools repo, https://github.com/istio/tools/blob/master/cmd/gen-release-notes/note.go#L67 it seems that we did not have any constraints on the notes area. So, in this PR's case, does `pilot` sounds okay to you? Or, we do want to add some areas constraints. Thanks!

Reply:
I just learn from others:
```
root:[notes]$ git grep "area: pilot"
26001.yaml:3:area: pilot
27976.yaml:3:area: pilot
```

Reply:
Yeah, if so, it means we did not have any constraints on `area`. I'm not sure if we want to add some or not. Let's wait the response from other reviewers.

Reply:
I see #28276.

Reply:
Release note should use one of the areas mentioned here: https://github.com/istio/istio/tree/master/releasenotes#area. While the release notes tool itself is generic, none of the templates currently include a field for Pilot. I think in this case, traffic-management is probably what you're looking for

Reply:
Updated.